src/mumble/VoiceRecorder.h

```diff
@@ -51,6 +51,8 @@ enum Format {
 	AU,
 	/// FLAC Format
 	FLAC,
+    // OPUS Format
+    OPUS,
```

Comment:
Wrong indentation (spaces instead of tabs). This also applies to some of your other changes.
Running clang-format on the source files you touched will resolve all formatting-related issues.

Reply:
The spaces are still there. Please run clang-format